tests/unit_tests/api/test_feature.py

```diff
@@ -90,7 +90,7 @@ def test_get_feature_toggle_failed_etag():
 
 
 @pytest.mark.skipif(
-    date.today() < date(2023, 10, 1),
+    date.today() < date(2023, 11, 1),
```

Comment:
I create this issue about it https://github.com/Unleash/unleash-client-python/issues/275